EIPS/eip-1227.md

```diff
@@ -0,0 +1,64 @@
+---
+eip: 1227
+title: Defuse Difficulty Bomb and Reset Block Reward
+author: SmeargleUsedFly (@SmeargleUsedFly)
+discussions-to: https://github.com/ethereum/EIPs/issues/1227
+status: Draft
+type: Standards Track
+category: Core
+created: 2018-07-18
+requires: 649
+---
+
+## Simple Summary
+This EIP proposes to permanently disable the "difficulty bomb" and reset the block reward to pre-Byzantium levels.
+
+## Abstract
+Starting with `FORK_BLKNUM` the client will calculate the difficulty without the additional exponential component. Furthermore, block rewards will be adjusted to a base of 5 ETH, uncle and nephew rewards will be adjusted accordingly.
+
+## Motivation
+Nick Johnson, the [de facto lead EIP editor](https://gitter.im/ethereum/AllCoreDevs?at=5b4e00d6fd1b3474a69834bc), has [stated unambiguously](https://old.reddit.com/r/ethereum/comments/8zk7l0/changes_are_being_made_to_the_eip_process_which/e2jl29o/) that the Ethereum user community's opinion on changes to the protocol may only be announced by running/not running node client software. This is also reflected in EIP [#1](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md), which includes no mention of community input over the EIP process. However, due to the "difficulty bomb" (also known as the "ice age"), introduced in EIP [#2](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2.md), an artificial exponential increase in difficulty until chain freeze, users may find it much more challenging to exercise one of those choices than the other&mdash;namely, remaining on the unforked chain after a controversial hard-fork.
```

Comment:
What purpose does this section serve in the EIP? It doesn't appear to be relevant to the technical merits of the proposal.

Reply:
It serves the purpose of framing the motivation for the EIP. What specific part of that paragraph is of concern? If it's the beginning, it should be noted that it is not intended as a sleight, but as a statement of fact on the current state of affairs. I'm a little confused on the expression "technical merits": what exactly do you mean by this?

Reply:
An EIP is accepted if it's technically sound. The motivation should describe why it's being written and why it should be implemented, but I don't think what I say in a Reddit post has any relevance to that at all. What's more, the actual content here - about EIP governance - has no impact on whether this EIP is sound either.

Reply:
Updated wording in a14405c